### PR TITLE
Small bug fixes in input of Excel file

### PR DESCRIPTION
**Pull request by N-thony**
Fixing a small bug in the input of the Excel file. This solves issue #6129
@africanmathsinitiative/developers and @rdstern this is ready for review. Thanks.

### CODE REVIEW

File: instat/dlgImportDataset.vb [hunk 1]
```diff
@@ -952,7 +953,8 @@ Public Class dlgImportDataset
                 ElseIf dctSelectedExcelSheets.Count = 1 Then
                     strSheetNumbers = dctSelectedExcelSheets.Keys.First()
                     clsImportExcel.AddParameter("which", strSheetNumbers)
-                    ucrSaveFile.SetName(dctSelectedExcelSheets.Values.First(), bSilent:=True)
+                    strSheetName = dctSelectedExcelSheets.Values.First()
+                    ucrSaveFile.SetName(strSheetName.Replace(" ", "."), bSilent:=True)
```
Comment (Patowhiz) [inline instat/dlgImportDataset.vb:957]: The changes made here, do indeed fix this bug.
@N-thony Also have a look at my implementation in PR #6065 code line 456 `GetCleanedName(strName As String)`, it relates to scenarios where such invalid "R names" are converted to valid "R names".

**Comment by Patowhiz** [comment]
@rdstern I tested the problem in my pending PR #6065 and the sheet name is renamed accordingly. You may want to also test PR #6065 thoroughly. It's meant to refactor the code in this dialog, but all existing functionalities are supposed to work.

@lloyddewit I'm not sure how long PR #6065 will be pending. If merged, it will solve the issue raised here.  This PR does indeed solve the specific bug but if merged first then I will have to resolve conflicts.

**Comment by lloyddewit** [comment]
@N-thony @Patowhiz Thank you for the bug fix and review respectively.
PR #6065 is following the normal process. A few days ago, I asked @shadrackkibet to review. It is quite a large PR and it will take some time to review. @shadrackkibet has lots to do at the moment but I will still keep reminding him about PR #6065 ;)
Roger will then test and I will then review and hopefully we can merge.

I suggest we wait for the PR #6065 merge. If this solves issue #6129 then we can close this PR.

**Review by rdstern — APPROVED** [review]
Now looks fine to me

File: instat/dlgImportDataset.vb (hunk 1 above)
Comment (lloyddewit) [reply instat/dlgImportDataset.vb:957]: @N-thony @Patowhiz The `GetCleanedName` function does a more robust 'cleaning'. It's a private function in another dialog so we can't use it directly. In a future refactoring we could consider having some utility functions that can be used by multiple classes. For now, I'll accept the change above.

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
Previously, we planned to close this PR manually after PR#6065 was merged. However, @rdsternspecifically asked me to merge this PR now so that @Patowhiz can continue with functionality for importing multiple files.

Please note also that in my comment above I incorrectly stated 'The `GetCleanedName` function ... is a private function in another dialog'. In fact it's in the same dialog but PR #6065 hasn't been merged yet.

**Comment by lloyddewit** [comment]
I just got a message from @rdstern: 'Ooops I got that wrong!  It was 6222 that was needed!'. So sorry for the confusion!

I think this PR has no negative impact so I will leave it merged.

**Pull request merged**